src/Controller/PageAdminController.php

```diff
@@ -253,16 +253,19 @@ public function composeContainerShowAction(Request $request): Response
         // filter service using the template configuration
         if (null !== $page) {
             $template = $this->container->get('sonata.page.template_manager')->get($page->getTemplateCode());
+            $code = $block->getSetting('code');
 
-            $container = $template->getContainer($block->getSetting('code'));
+            if (null !== $code) {
```

Comment:
`getContainer` function does not acept null, we need to protect with an if, because getSetting can return null.